deployment/configutils/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ include_directories (
          ${HPCC_SOURCE_DIR}/deployment/deploy
          ${CMAKE_BINARY_DIR}/oss
          ${CMAKE_BINARY_DIR}
+         ${HPCC_SOURCE_DIR}/system/security/shared
```

Comment:
Evolution I suppose, and it should be fixed. But that's beyond the scope of this PR